src/webview/debuggerPanel.ts

```diff
@@ -63,27 +63,33 @@ export class DebuggerPanel {
 
   updatePanel(panelViewInput: PanelViewInput): void {
     this.currentPanelViewInput = panelViewInput;
-    if (this.viewPanel !== undefined && panelViewInput !== undefined) {
-      void this.viewPanel.webview.postMessage(this.panelViewProxy.updatePanel(panelViewInput));
+    if (panelViewInput !== undefined) {
+      this.postCommandToWebViewIfViewPanelIsDefined(this.panelViewProxy.updatePanel(panelViewInput));
     }
   }
 
   exportPanel(): void {
-    if (this.viewPanel !== undefined) {
-      void this.viewPanel.webview.postMessage(this.panelViewProxy.exportPanel());
-    }
+    this.postCommandToWebViewIfViewPanelIsDefined(this.panelViewProxy.exportPanel());
   }
 
   startRecordingPanel(): void {
-    if (this.viewPanel !== undefined) {
-      void this.viewPanel.webview.postMessage(this.panelViewProxy.startRecordingPanel());
-    }
+    this.postCommandToWebViewIfViewPanelIsDefined(this.panelViewProxy.startRecordingPanel());
   }
 
   stopRecordingPanel(): void {
-    if (this.viewPanel !== undefined) {
-      void this.viewPanel.webview.postMessage(this.panelViewProxy.stopRecordingPanel());
-    }
+    this.postCommandToWebViewIfViewPanelIsDefined(this.panelViewProxy.stopRecordingPanel());
+  }
+
+  exportAsPlantUml(): Promise<void> {
+    return this.panelViewProxy.exportAsPlantUml();
```

Comment:
Don't send this to the proxy, as mentioned [here](https://github.com/GinoCardillo-OST/Visual-OO-Debugger/pull/24/files#r775040063).

Reply:
Done.